docs/website/netcdf-java/reference/stream/CdmrCommon.adoc

```diff
@@ -1,12 +1,12 @@
 CDM Remote is a web service implemented in the CDM library (client) and TDS (server), providing remote access to CDM datasets, using
-<<NcStream#,ncstream>> as the on-the-wire protocol.
+link:NcStream.adoc[ncstream] as the on-the-wire protocol. An experimental python client is under development.
 It provides access at the NetcdfFile and FeatureDataset levels of the CDM API, so there are two levels of services:
 
-.  *<<CdmRemote#,CDM Remote>>* provides index subsetting on remote CDM datasets, with the same functionality that *NetcdfFile* provides for
+.  *link:CdmRemote.adoc[CDM Remote]* provides index subsetting on remote CDM datasets, with the same functionality that *NetcdfFile* provides for
 CDM local datasets and *DODSNetcdfFile* provides for remote OPeNDAP datasets. CdmRemote supports the full CDM data model.
-.  *<<CdmrFeatureGrid#,CDM Remote Feature for Coverage/Grid Data>>* provides coordinate subsetting on remote CDM Coverage (Grid) Feature Datasets, with similar functionality to WCS and
-Unidata's <<{tdsDocs}/NetcdfSubsetServiceReference#,NetCDF Subset Service>> (NCSS).
-.  *<<CdmrFeaturePoint#,CDM Remote Feature for Point Data>>* provides coordinate subsetting on remote CDM Point (DSG) Feature Datasets, with similar functionality to
-Unidata's <<{tdsDocs}/NetcdfSubsetServiceReference#,NetCDF Subset Service>> (NCSS).
+.  *CDM Remote Feature* provides coordinate subsetting on remote CDM Feature Datasets, with similar functionality to OGC's
+*Web Coverage Service* and the exact functionality of Unidata's *link:{tdsDocs}/NetcdfSubsetServiceReference.adoc[NetCDF Subset Service]* (NCSS).
+..  link:CdmrFeatureGrid.adoc[CDM Remote Feature for Coverage/Grid Data] provides coordinate subsetting on remote CDM Coverage (Grid) Feature Datasets
+..  link:CdmrFeaturePoint.adoc[CDM Remote Feature for Point Data] provides coordinate subsetting on remote CDM Point (DSG) Feature Datasets
```

Comment:
All of these should be cross-references. Rule of thumb: we should never link to `.adoc` files.